server/routes/irs.js

```diff
@@ -6,4 +6,19 @@ router.get('/', function (req, res) {
   res.send(irsObj);
 });
 
+router.post('/:status', function (req, res){
+  var newStatus;
+  if (req.params.status === '10'){
+    newStatus = '11';
+  } else {
+    newStatus = '10';
+  }
+  res.status(202).send({
+    status: {
+      code: newStatus,
+      message: ""
```

Comment:
We _could_ also update with a message to help the front-end. Probably not necessary, at least not yet.